### PR TITLE
Style/abd style unification

### PR DESCRIPTION
**Pull request by Abdeboskey**
# Som Timer Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
* This branch attempts to unify the styling across the entire application
* Adjust spacing, colors, shadows, font-sizes, to accomplish the above goal.
* Add custom icons for navbar in header, timer controls, and content selection buttons

## Type of change
* _Style_

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
* This has been tested visually and with the inspector in the devtools.

# Message for reviewer:
* We are currently missing an Icon for 'Yoga/Movement' in the content selection, which is being developed and will be added soon.
* There is a bug on the content delivery section that is creating a mystery margin-right on the video player, and on the header in mobile view.

* **Please Let us know if there are any details/things that you would like to see differently!!**

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

TimerView
ContentSelectionView
StatsComingSoon
AboutViewTop
AboutViewBottom
SettingsView
NotFound

### CODE REVIEW

**Review by rwilliams659 — APPROVED** [review]
Thanks so much for working so hard on styling, this is looking awesome!

File: src/Components/About/About.test.js [hunk 1]
```diff
@@ -8,18 +8,18 @@ describe('About', () => {
...
-    const frontEndHeading = getByRole('heading', { name: 'Front End Engineers:'})
-    const frontEndEngineer1 = getByText('Aaron Burris-DeBoskey')
-    const frontEndEngineer2 = getByText('Jake West')
-    const frontEndEngineer3 = getByText('Rachel Williams')
-    const backEndHeading = getByRole('heading', { name: 'Back End Engineers:' })
-    const backEndEngineer1 = getByText('Chandler Hulstrom')
-    const backEndEngineer2 = getByText('Dorion')
-    const backEndEngineer3 = getByText('Sienna Kopf')
+    const aboutHeading = getByRole('heading', { name: /about som timer/i })
+    const appDescription1 = getByText(/fight cognitive boredom and burnout/i)
```
Comment (jkwest-93) [inline src/Components/About/About.test.js:12]: The regex magic you mentioned!! Today I learned!

File: src/Components/Header/Header.module.scss [hunk 2]
```diff
@@ -36,18 +38,34 @@
...
 }
 
+.settingsButton {
+  @extend .navButton;
+  &:hover {
+    animation: infinite-spinning 3s ;
+  }
+}
+
+@keyframes infinite-spinning {
```
Comment (jkwest-93) [inline src/Components/Header/Header.module.scss:60]: I love this animation! Great work here.

File: src/Components/About/About.test.js (hunk 1 above)
Comment (Abdeboskey) [reply src/Components/About/About.test.js:12]: 🔮

**Review by jkwest-93 — APPROVED** [review]
Great work with this aesthetically! I really enjoy learning new CSS tricks from you, as well as the Regex testing tip for text.

**Review by sciencefixion — APPROVED** [review]

**Pull request merged**